scripts/_common.sh

```diff
@@ -6,8 +6,6 @@
 
 YNH_PHP_VERSION="8.1"
```

Comment:
I would tend to get rid of this line, the `phpversion` setting automatically defined during the `apt` step should be enough

```suggestion
```